Makefile

```diff
@@ -1,132 +1,36 @@
-# Shortcut targets
-default: build
-
-## Build binary for current platform
-all: build
-
-## Run the tests for the current platform/architecture
-test: ut
+PACKAGE_NAME?=github.com/projectcalico/node
```

Comment:
Why is this `node`, for my make benefit?

Reply:
That's a bug :-) Fixed